website/docs/r/aws_auth_backend_roletag_blacklist.html.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_auth_backend_roletag_blacklist resource"
+sidebar_current: "docs-vault-resource-aws-auth-backend-roletag-blacklist
+description: |-
+  Configures the periodic tidying operation of the blacklisted role tag entries.
+---
+
+# vault\_aws\_auth\_backend\_roletag\_blacklist
+
+Configures the periodic tidying operation of the blacklisted role tag entries.
+
+## Example Usage
+
+```hcl
+resource "vault_auth_backend" "example" {
+  type = "aws"
+}
+
+resource "vault_aws_auth_backend_roletag_blacklist" "example" {
+  backend       = "${vault_auth_backend.example.path}"
+  safety_buffer = 360
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Required) The path the AWS auth backend being configured was
+	mounted at.
+
+* `safety_buffer` - (Oprtional) The amount of extra time that must have passed
```

Comment:
Typo for `Optional` and it might be worth mentioning the time unit here, e.g. `extra time (in seconds)`